### PR TITLE
Adding MultiApps bot

### PR DESCRIPTION
**Pull request by ikasarov**
Adding a new bot github account for use with release/branch restrictions

### CODE REVIEW

**Comment by beyhan** [comment]
Hi @ikasarov,

two approvers of the MultiApps area need to approve this pr or put thumps up on it before merging.

**Review by stephanme — CHANGES_REQUESTED** [review]

**Review by boyan-velinov — APPROVED** [review]

**Review by theghost5800 — APPROVED** [review]

**Review by s-yonkov-yonkov — APPROVED** [review]

File: org/contributors.yml
Comment (ikasarov) [reply org/contributors.yml]: removed it

**Review by stephanme — APPROVED** [review]

**Pull request merged**
